microsoft-teams/1.0/modules/ROOT/pages/microsoft-teams-connector-examples.adoc

```diff
@@ -0,0 +1,427 @@
+= Microsoft Teams Connector 1.0 Examples - Mule 4
+
+These examples shows you how to use Microsoft Teams Connector to perform the following actions:
+
+* <<create-new-team,Create a new team>>
+* <<retrieve-team,Retrieve team properties and relationships>>
+* <<retrieve-membership,Retrieve a list of team members>>
+* <<retrieve-channel-list,Retrieve a list of team channels>>
+* <<onboard-new-hire,Onboard a new hire to the team>>
+* <<delete-channel,Delete a team channel>>
+* <<remove-team-member,Remove a team member>>
+
+== Prerequisites
+
+To run these examples, you must have:
+
+* A working instance of Microsoft Teams
+* Credentials to access the Microsoft Teams instance
+* Microsoft Teams Connector 1.0 or later
+* Two users in the Microsoft Teams instance: one team owner and one user
+* Anypoint Studio 7.x
+
+== Configure OAuth Authorization
+
+Because this connector uses OAuth to secure connections, you must specify OAuth field values, such as client credentials, keys, and OAuth-related URLs. To do this:
+
+* Create a configuration file for the OAuth Connection
+* Configure the OAuth-related global elements
+* Configure a global element for the properties file
+
+=== Create a Configuration File for the OAuth Connection
+
+Create a configuration file for the OAuth connection:
+
+. Create a file named `mule-app.properties` in the `/src/main/resources/` folder.
+. In the `mule-app.properties` file, create a set of properties for `Authorization Code Grant Type` like the ones that follows. Overlay the bracketed text (including the brackets) with the correct values for your OAuth configuration:
```

Comment:
```suggestion
. In the `mule-app.properties` file, create a set of properties for `Authorization Code Grant Type`, similar to the ones that follow, overlaying the bracketed text (including the brackets) with the correct values for your OAuth configuration:
```

Reply:
I often substitute "similar to" for "like" for translation/globalization considerations.